.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
```

Comment:
How about the rest client extension?

Reply:
I wasn't supposed to add this in the PR, so I deleted it.